CONTRIBUTING.md

```diff
@@ -27,3 +27,6 @@ Here are a few things that will help us help resolve your issues:
 - Push to your branch and PR
 - Wait for your pull request to be reviewed and merged!
 - PRs that exclusively add items to the list of domains will be classed as spam/invalid. We already grab this list from ICANN's complete list of certified registrars.  
+
+Note:
+If the PR you have made is considered to be a major one, once it has the `enhancement` label, please increase the version of the extension in [manifest.json](https://github.com/immattdavison/NoMoreDomains/blob/master/manifest.json#L4).
```

Comment:
`'a major one'` should maybe be reworded to `'a major update'`?

Reply:
Maybe `If the PR you have made contains a major update`? This makes more grammatical sense to me. Its up to you

Reply:
> Maybe `If the PR you have made contains a major update`? This makes more grammatical sense to me. You decide

Yes.